backend/lib/endpoints/users.js

```diff
@@ -37,6 +37,17 @@ async function routes(app) {
       };
     },
   );
+
+  app.post(
```

Comment:
So, I think this should change to `POST /api/users` since creating a new POST should not include the id
usually this id is created in the backend, like in the mongo collection
but in this case we get it from the JWT in the `mongo_id` property that we added in Auth0

I think for now the only thing that needs to be added too is that we make sure the user has a valid email address in Auth0, so we can use probably email from `req.user` I think, but email_verified might need an additional request to `Auth0.getUser()`, alternatively we could also add it to the JWT... can you work on this @pishili ?